scripts/build-nuget-packages.sh

```diff
@@ -10,3 +10,6 @@ mkdir -p ./nupkgs
 # For simplicity, some packages depend on Improbable.WorkerSdkInterop. Make sure that's packaged first.
 dotnet pack Improbable/WorkerSdkInterop/Improbable.WorkerSdkInterop -p:Platform=x64 --output "$(pwd)/nupkgs"
 dotnet pack Improbable -p:Platform=x64 --output "$(pwd)/nupkgs"
+
+# Beep.
+echo -ne '\007'
```

Comment:
Nit - trailing new line

Reply:
Also TIL - does this work cross platform?

Reply:
Yep! As long as your PC has a way of making sound